hikari/impl/special_endpoints.py

```diff
@@ -693,6 +696,33 @@ async def __anext__(self) -> audit_logs.AuditLog:
         return log
 
 
+@attr_extensions.with_copy
+@attr.define(kw_only=False, weakref_slot=False)
+class InteractionAutocompleteBuilder(special_endpoints.InteractionAutocompleteBuilder):
+    """Standard implementation of `hikari.api.special_endpoints.InteractionAutocompleteBuilder`."""
+
+    _choices: typing.Sequence[commands.CommandChoice] = attr.field(factory=list)
+
+    @property
+    def choices(self) -> typing.Sequence[commands.CommandChoice]:
+        return self._choices
+
+    def set_choices(
+        self: _InteractionAutocompleteBuilderT, choices: typing.Sequence[commands.CommandChoice], /
+    ) -> _InteractionAutocompleteBuilderT:
+        """Set autocomplete choices
+
+        Returns
+        -------
+        InteractionAutocompleteBuilder
+            Object of this builder.
+        """
+        self._choices = choices
+
+    def build(self, _: entity_factory_.EntityFactory, /) -> data_binding.JSONObject:
+        return {"choices": self._choices}
```

Comment:
This is not correct, since it will map to the object. For contrast, in the RESTImpl you did:

```py
data.put("choices", [{"name": choice.name, "value": choice.value} for choice in choices])
```